fingerprint2.js

```diff
@@ -428,6 +460,23 @@
         done(keys);
       }, 1);
     },
+    mediaDevicesKey: function(keys, done) {
+        //Adds the device key for each media device to the devicesArr
+        if(navigator.mediaDevices && !this.options.excludeMediaDevices){
+            var devicesArr = [];
+            var that = this;
+            navigator.mediaDevices.enumerateDevices().then(function(devices){
```

Comment:
Media devices are only supported in Chrome 45+ with a flag, so in reality they are not supported currently in all major browsers, please see this: https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices

Please add error handling for cases when it's not supported. I believe current code will raise errors in browsers that don't support them.

You're also using promise here, so this ticket should be blocked by #13